react/src/Searchbar/Searchbar.tsx

```diff
@@ -16,12 +16,18 @@ import {
 import { IconButton } from '~/IconButton'
 import { BxSearch, BxX } from '~/icons'
 
-export interface SearchbarProps extends InputProps {
+export interface SearchbarProps extends Omit<InputProps, 'onChange'> {
   /**
    * Function to be invoked when user presses enter (to search).
    * @param searchValue value of the search input
    */
-  onSearch: (searchValue: string) => void
+  onSearch?: (searchValue: string) => void
+
+  /**
+   * Function to be invoked when the search input has changed.
+   * @param searchValue value of the search input
+   */
+  onChange?: (searchValue: string) => void
```

Comment:
do we need a new function? I think it should be ok enough to make onSearch optional if that is the case. Users of the component can pass in their own `onChange` function to react to changes, no?

Reply:
Sounds reasonable, updated in [0c38dc7](https://github.com/opengovsg/design-system/pull/218/commits/0c38dc7ef24adc52c9febcc8e0102375630230e8)